### PR TITLE
working nightly with fastlane

### PR DESCRIPTION
**Pull request by rasom**
status: ready

### CODE REVIEW

File: Jenkinsfile.nightly [hunk 1]
```diff
@@ -32,17 +32,16 @@ node ('macos1'){
...
+      latest_tag = sh(returnStdout: true, script: 'git describe --tags `git rev-list --tags --max-count=1`').trim()
+      sh 'git tag -d ' + latest_tag
       sh 'rm -rf node_modules'
       sh 'cp .env.jenkins .env'
       sh 'lein deps'
       installJSDeps()
 
       sh 'mvn -f modules/react-native-status/ios/RCTStatus dependency:unpack'
-        sh 'cd ios && pod install && cd ..'
+        sh 'cd ios &&  pod install && cd ..'
```
Comment (yenda) [inline Jenkinsfile.nightly]: does the extra space change anything here ?

File: Jenkinsfile.nightly [hunk 2]
```diff
@@ -58,19 +57,21 @@ node ('macos1'){
     }
 
     stage('Build (iOS)') {
+    withCredentials([string(credentialsId: 'jenkins_pass', variable: 'password')]) {
```
Comment (yenda) [inline Jenkinsfile.nightly]: a 2 space indent would make things clearer

**Review by yenda — APPROVED** [review]

**Review by flexsurfer — APPROVED** [review]

File: Jenkinsfile.nightly (hunk 1 above)
Comment (rasom) [reply Jenkinsfile.nightly]: i doubt :D

File: Jenkinsfile.nightly (hunk 2 above)
Comment (rasom) [reply Jenkinsfile.nightly]: sure

**Comment by Serhy** [comment]
If there is a same git revision that job has been run against - the job fails with the 
`Google Api Error: apkUpgradeVersionConflict: APK specifies a version code that has already been used.`
We need to handle the case if there no changes in code was since last successful job ran (maybe just skip to exec fastlane in that case?)

**Comment by rasom** [comment]
@Serhy will be fixed in separate PR

**Pull request merged**